src/cmd/purge/scripts/exchangePurge.ps1

```diff
@@ -536,6 +543,12 @@ function Purge-Contacts {
         
         }
     }
+
+    # Create one seed contact which will have recent create date and will not be swept
+    # This is needed since tests rely on some contact data being present
+
+    Write-Host "`nCreating seed contact" 
+    Create-Contact
```

Comment:
@ryanfkeepers the previous location of this was actually safer since the new contact was in place before the contacts were purged. Prevents a case where a test that relies on at least 1 contact being present executes in the timing window where all contacts were purged and the new one is not there.

Reply:
I can fix this for you and will approve it

Reply:
Done

Reply:
much appreciated